linux_os/guide/services/obsolete/tftp/tftp_uses_secure_mode_systemd/ansible/shared.yml

```diff
@@ -0,0 +1,32 @@
+# platform = multi_platform_all
+# reboot = false
+# complexity = low
+# strategy = configure
+# disruption = low
+
+{{{ ansible_instantiate_variables("var_tftpd_secure_directory") }}}
+
+-   name: "{{{ rule_title }}} - Check if tftp.service exists"
+    ansible.builtin.stat:
+        path: "/usr/lib/systemd/system/tftp.service"
+    register: tftp_service_file
+
+-   name: "{{{ rule_title }}} - Ensure ExecStart includes secure directory"
+    community.general.ini_file:
+        path: "/usr/lib/systemd/system/tftp.service"
+        section: Service
+        option: ExecStart
+        value: "/usr/sbin/in.tftpd -s {{ var_tftpd_secure_directory }}"
```

Comment:
Editing system files is something that I'm not a huge fan of. If we edit this file I believe we will cause rpm check to fail.

 I would prefer to use drop-in files, but auditing that with OVAL gets tricky. However, we might just be able to add the drop path as check as a compromise.